content/how-to/rackspace-fabric/Notification Preferences/index.md

```diff
@@ -0,0 +1,25 @@
+---
+permalink: "Notification Preferences"
+audit_date: '2022-03-24'
+type: article
+title: "Notification Preferences"
+created_date: '2022-03-24'
+created_by: Rohit Rajput
+last_modified_date: '2022-03-24'
+last_modified_by: Rohit 
+product: Rackspace Fabric
+product_url: rackspace-fabric
+---
+Managing Notifications
+================================
+**Step 1**. Login to https://portal.rackspace.com.
```

Comment:
It seems like our applications are trying to get off of portal.rackspace.com . They have been moving over to https://manage.rackspace.com
We might as well update this to https://manage.rackspace.com already.